lib/socketServer.js

```diff
@@ -41,7 +42,17 @@ module.exports = (app, { verbose }) => client => {
   if (verbose) io.use(log())
 
   const subscriptions = new Set()
-  io.state = { client, subscriptions }
+  const serviceType = mdns.makeServiceType('socketio', 'tcp', 'livingroom')
```

Comment:
in the `httpServer` you are calling this method differently. 

```js
const serviceType = mdns.makeServiceType(['socketio', 'tcp', 'livingroom'])
// vs.
const serviceType = mdns.makeServiceType('socketio', 'tcp', 'livingroom')
```

seems like a possible bug :)